.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,97 @@
+name: Bug Report
+description: File a bug report
+labels: ["t/bug"]
+assignees: []
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report! Please make sure to add as much detail as you can, preferably with a reproduction. This will help us diagnose the issue faster and thus resolve it quicker.
+        
+        If this is a bug in Xamarin.Forms, please [open the issue in the Xamarin.Forms repo](https://github.com/xamarin/Xamarin.Forms/issues/new?assignees=&labels=s%2Funverified%2C+t%2Fbug+%3Abug%3A&template=bug_report.md&title=[Bug]+).
+  - type: textarea
+    id: description
+    attributes:
+      label: Description
+      description: Please give us a detailed description of the issue that you're seeing. You can add screenshots, videos and reproduction projects (zipped) here as well.
```

Comment:
This line might need to be updated. I sent an email including you @jfversluis so please wait for that to resolve before merging.